core/app/[locale]/(default)/page.tsx

```diff
@@ -58,7 +58,7 @@ export default async function Home({ params }: Props) {
 
   return (
     <>
-      <Slideshow />
+      <HomepageSlideshow />
```

Comment:
This is something that I wasn't sure wether to suggest or not @jorgemoya. I'll delegate to you here since I'm sure you guys already discussed this at length in the past. Is the idea to scope components as much as possible under `_components` in `app`?

How is the decision to pull it out to the top-level `components` dir made? Why not have all components under `app/_components`?

Reply:
Something else. Even if we scope it down to be under `app/[locale]/(default)/_components` what do you think if we keep the `HomepageSlideshow` name? I personally prefer the verbosity since if we end up having another `Slideshow` component, either as a primitive in Soul, a section in Soul (which we actually already have) or as a slideshow component in another route, it gets very annoying with VS Code auto-completes.

Not a super strong opinion here, but generally I prefer globally unique names for things that are exported from modules.

Curious about your opinion on this one @deini

Reply:
The general rule is that if the component is being reused we add it globally (ie. ~/components) otherwise we try to position as close as possible to where it's being used. For me, it provides a clear picture of what the page is doing. I believe this is one of the issues of having section components in VIBES (or adding everything in ~/vibes), there's a big jump between contexts when developing a page.

Regarding the name, I still prefer the <Slideshow /> name but not super against renaming.